Funcky.Async/Extensions/AsyncEnumerableExtensions/Shuffle.cs

```diff
@@ -1,17 +1,32 @@
-using static Funcky.Internal.Mixer;
+using Funcky.Internal;
 
 namespace Funcky.Async.Extensions
 {
     public static partial class AsyncEnumerableExtensions
     {
         /// <summary>
-        /// Returns the given sequence in random Order in O(n).
+        /// Returns the given sequence eagerly in random Order in O(n).
         /// </summary>
         /// <typeparam name="TSource">The type of the elements in the enumerable.</typeparam>
         /// <remarks>This method is implemented by using deferred execution. The immediate return value is an object that stores all the information that is required to perform the action. The query represented by this method is not executed until the object is enumerated either by calling its GetEnumerator method directly or by using foreach.</remarks>
         [Pure]
-        public static async ValueTask<IEnumerable<TSource>> Shuffle<TSource>(this IAsyncEnumerable<TSource> source, CancellationToken cancellationToken = default)
+        public static async ValueTask<IEnumerable<TSource>> Shuffle<TSource>(this IAsyncEnumerable<TSource> source)
```

Comment:
Shouldn't the return type be `ValueTask<IReadOnlyList<..>>`?

Reply:
Yes, that was a bad oversight.

Reply:
Fixed in d43e902a8f37df1c8d415a140316817707b84cfa